cypress/e2e/settings.cy.js

```diff
@@ -1,32 +1,69 @@
 /// <reference types="cypress" />
 /// <reference types="../support" />
+import SignInPageObject from '../support/pages/signIn.pageObject';
+import homePageObject from '../support/pages/home.pageObject';
+import SettingsPageObject from '../support/pages/settings.pageObject';
+const faker = require('faker');
 
-describe('Settings page', () => {
-  before(() => {
+const signInPage = new SignInPageObject();
+const homePage = new homePageObject();
+const settingsPage = new SettingsPageObject();
 
-  });
+describe('Settings page', () => {
+   let user;
 
   beforeEach(() => {
-
+   cy.task('db:clear');
+    cy.task('generateUser').then((generateUser) => {
+      user = generateUser;
+    cy.login(user.email, user.username, user.password);
+      settingsPage.visit();
+    });
   });
 
   it('should provide an ability to update username', () => {
+   const newUsername = faker.name.firstName().toLowerCase();
+
+   settingsPage.fillUsernameField(newUsername);
+   settingsPage.clickUpdateBtn();
 
+   homePage.assertHeaderContainUsername(newUsername);
   });
 
   it('should provide an ability to update bio', () => {
+   const newBio = faker.lorem.words();
 
+   settingsPage.fillUserBioField(newBio);
+   settingsPage.clickUpdateBtn();
+   homePage.assertUpdatedBio(newBio);
   });
 
   it('should provide an ability to update an email', () => {
+   const newEmail = faker.internet.email();
+   settingsPage.fillEmailField(newEmail);
+   settingsPage.clickUpdateBtn();
 
+   homePage.assertHeaderContainUsername(user.username);
```

Comment:
how do you check that email has been updated using this method?
```suggestion
   homePage.assertHeaderContainUsername(user.username);
```